### PR TITLE
default patterns in custom roles

### PR DESCRIPTION
**Pull request by schubi2**
This should be submitted together with
https://github.com/yast/yast-installation/pull/568
But also runs without it.

### CODE REVIEW

**Comment by jreidinger** [comment]
can you link here also related pull request for packager or installer, that do such evaluation of patterns? in general this one looks good, but I would like to see rest.

**Comment by schubi2** [comment]
Screen shots are here:
https://trello-attachments.s3.amazonaws.com/5502d5dd8eb45fb4581c1a0f/591c12d2d776d75ac5a143c1/58b2e1735532fea1a22721f6ea6ec8f6/customer_rules.pdf

File: package/skelcd-control-openSUSE.changes [hunk 1]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jun  1 13:21:39 CEST 2017 - schubi@suse.de
+
+- control.cml: "custom" role has an own pattern selection screen
+  mask while the installation workflow. The default values can be
```
Comment (lslezak) [inline package/skelcd-control-openSUSE.changes]: It is hard to understand, I'd suggest something like this for the whole changelog:

*The "custom" role displays manual the pattern selection dialog in the installation workflow. The default pre-selected patterns can be configured in the "control.xml" file in the "system_role" section.*

File: control/control.openSUSE.xml [hunk 2]
```diff
@@ -418,6 +418,13 @@ textdomain="control"
 
       <system_role>
         <id>custom</id>
+        <software>
+          <!-- "custom" role has an own pattern selection screen mask while
```
Comment (lslezak) [inline control/control.openSUSE.xml]: Again, here I'd write this:

*The "custom" role displays manual the pattern selection dialog in the installation workflow.*

File: package/skelcd-control-openSUSE.changes [hunk 3]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jun  1 13:21:39 CEST 2017 - schubi@suse.de
+
+- control.cml: "custom" role has an own pattern selection screen
```
Comment (Vogtinator) [inline package/skelcd-control-openSUSE.changes]: s/cml/xml/

**Pull request merged**